### ISSUE TITLE
Accessibility (major): Landing page - Image alt text decorative book header banner [prea-001]

### ISSUE THREAD

**Issue by aaron-collier** [sul-dlss/pre-assembly#1256]
## Description

The decorative bookheader banner should have `<img alt="">` (i.e., empty alt) so that it is skipped by screen readers.

## Suggestion

Add `alt=""` to the `<image>`.

https://docs.google.com/spreadsheets/d/105GozrKuWrnXaEk5FoP4UbzHSCTPgeEqPaPiHi1c950/edit#gid=0
